flink-runtime/src/main/java/org/apache/flink/runtime/rest/handler/legacy/CurrentJobsOverviewHandler.java

```diff
@@ -92,24 +101,26 @@ public CurrentJobsOverviewHandler(
 					StringWriter writer = new StringWriter();
 					try {
 						JsonGenerator gen = JsonFactory.JACKSON_FACTORY.createGenerator(writer);
+						final JobDetails.JobDetailsSerializer jobDetailsSerializer = new JobDetails.JobDetailsSerializer();
+
 						gen.writeStartObject();
 
 						if (includeRunningJobs && includeFinishedJobs) {
 							gen.writeArrayFieldStart("running");
```

Comment:
"running" --> can use `MultipleJobsDetails.FIELD_NAME_JOBS_RUNNING`

Reply:
True, will change it.